.github/workflows/deploy-testnet.yml

```diff
@@ -37,6 +37,7 @@ jobs:
         run: |
           pnpm run build
           pnpm run build:inject-amplitude
+          pnpm run build:inject-hotjar
```

Comment:
fyi that this workflow is no longer used!

Reply:
shall I then drop this change?